### ISSUE TITLE
Adding Social Share links to govCMS8

### ISSUE THREAD

**Issue by tobybellwood** [govCMS/govcms8-installer#9]
Following on from discussion in https://community.govcms.gov.au/t/replacement-for-service-links-in-govcms-8/1064/10 

It seems that Service Links is a way off being Drupal 8 ready (https://www.drupal.org/project/contrib_tracker/issues/2606924) but Easy Social is (https://www.drupal.org/project/easy_social), and covers our needs (as gov publishers)

We should consider adding Easy Social to govCMS8.

**Comment by larowlan** [comment]
Easy social would violate most government agency privacy policies as it embeds third-party javascript for each service into the page, enabling 'dial home' so that these services can track what sites their users are visiting.
I'd be a solid -1 on easy social.
And a +1 on govcms organising a virtual sprint afternoon to work on service links, which supports link-based sharing, ie. no dial-home javascript.

**Comment by tobybellwood** [comment]
Thanks @larowlan - that's great feedback.  I'll look into the feasibility of a service_links approach!

**Comment by NigelCunningham** [comment]
@tobybellwood - I've started some work on service_links this morning. You can find it at https://github.com/NigelCunningham/service_links.

**Comment by larowlan** [comment]
@NigelCunningham we were talking about rebooting contrib office-hours on a Friday afternoon, where people with tricky d8 port problems could ask questions. That was normally 1pm on a Friday in Syd/Melb in #drupal-au on IRC. But given the volume of people in #australia on drupal.slack.com (http://drupalslack.herokuapp.com/ for an invite) I think it makes sense to reboot it there.

tl;dr if you have any questions on stuff, there's helpful folks there you can ping with questions - I for one are happy to help too. So ping me if you want reviews etc.

**Comment by NigelCunningham** [comment]
Thanks for the offer. It doesn't look too tricky so far. I'll certainly appreciate a review when it's ready - and I wasn't aware of the slack channel; will sign up!

**Comment by gargsuchi** [comment]
As per d.o - this is an option - https://github.com/HongPong/service_links We can certainly try and build on top of this.
https://www.drupal.org/project/contrib_tracker/issues/2606924

**Comment by NigelCunningham** [comment]
https://gitlab.com/nigelcunningham/service_links - contains a lot more work I did about this time last year.

**Comment by gargsuchi** [comment]
Some more options - 

- https://www.drupal.org/project/social_media
- https://www.drupal.org/project/addtoany
- https://www.drupal.org/project/social_media_links
- https://www.drupal.org/project/sharethis_block
- https://www.drupal.org/project/simplesharer

**Comment by simesy** [comment]
Based on the status of other new module requests, marking this as wontfix as it can be done relatively easily in the theme.  There is a new module request template also https://github.com/govCMS/govCMS8/issues/new/choose
